src/pages/SudokuPage.js

```diff
@@ -28,9 +31,9 @@ export default function SudokuPage() {
 
   function handleNumberInputClick(value) {
     if (selectedCell.length === 2) {
-      let newNumbArr = [...numbers];
+      let newNumbArr = [...sudoku];
```

Comment:
maybe  `newSudoku` or `nextSudoku` would be a better name? Just to be consistent

Reply:
Ah I changed it, but now I saw that I didn't saved the file before committing 🙄